proto/google/events/cloud/audit/v1/LogEntryData.json

```diff
@@ -1,387 +1,426 @@
-                    "items": {
-                        "type": "string",
-                        "examples": [
-                            "europe-west1-a",
-                            "us-east1",
-                            "nam3"
-                        ]
-                    }
-                },
-                "original_locations": {
-                    "description": "The locations of a resource prior to the execution of the operation. Requests that mutate the resource's location must populate both the 'original_locations' as well as the 'current_locations' fields. For example:",
-                    "type": "array",
-                    "items": {
-                        "type": "string",
-                        "examples": [
-                            "europe-west1-a",
-                            "us-east1",
-                            "nam3"
-                        ]
-                    }
-                }
-            }
+  "$id": "google.events.cloud.audit.v1",
+  "$schema": "http://json-schema.org/schema",
+  "title": "Cloud Audit Log LogEntry v1",
+  "name": "LogEntryData",
+  "description": "This event is triggered when a new audit log entry is written.",
+  "type": "object",
+  "goPackage": "auditv1",
+  "definitions": {
+    "MonitoredResource": {
+      "type": "object",
+      "description": "The monitored resource that produced this log entry.",
+      "properties": {
+        "type": {
+          "description": "Required. The monitored resource type. For example, the type of a Compute Engine VM instance is `gce_instance`.",
+          "type": "string"
         },
-        "Status": {
-            "type": "object",
-            "properties": {
-                "code": {
-                    "type": "integer"
-                },
-                "message": {
-                    "type": "string"
-                },
-                "details": {}
-            }
+        "labels": {
+          "description": "Values for all of the labels listed in the associated monitored resource descriptor. For example, Compute Engine VM instances use the labels `\"project_id\"`, `\"instance_id\"`, and `\"zone\"`.",
+          "type": "object"
+        }
+      }
+    },
+    "ResourceLocation": {
+      "type": "object",
+      "$comment": "https://cloud.google.com/service-infrastructure/docs/service-control/reference/rpc/google.cloud.audit?hl=en#resourcelocation",
+      "description": "Location information about a resource.",
+      "properties": {
+        "current_locations": {
+          "description": "The locations of a resource after the execution of the operation. Requests to create or delete a location based resource must populate the 'current_locations' field and not the 'original_locations' field.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "examples": [
+              "europe-west1-a",
+              "us-east1",
+              "nam3"
```

Comment:
Not in scope of this PR, but what is nam3? Examples are ideally usable, not just syntactically valid.

Reply:
This schema is created from the proto source

https://github.com/googleapis/google-cloudevents/blob/f9bb283233e2834034396c745c3600954cef7052/proto/google/events/cloud/audit/v1/data.proto#L337